packages/python-packages/api-stub-generator/apistub/_apiview.py

```diff
@@ -36,6 +37,7 @@ def __init__(self, nodeindex, pkg_name="", pkg_version=0, ver_string=""):
         self.Navigation = []
         self.Diagnostics = []
         self.indent = 0
+        self.namespace = namespace
```

Comment:
I can't see where namespace is actually used?

Reply:
I was planning to use this when checking if type name is part of current package or not. Currently it is not used but I expect this will be useful in near future.